packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/FlowsheetDataGrid.4/Examples.6/FlowsheetSectionsWithColHighlight.doc.mdx

```diff
@@ -16,3 +16,5 @@ This example demonstrates a [Flowsheet Data Grid](/components/cerner-terra-frame
 - `columnHighlightDescription` can be used to provide context regarding the highlighted column to screen readers.
 
 <FlowsheetSectionsWithColHighlight />
+
+Note: Column Highlighting functionality will not be supported for columns with column span. 
```

Comment:
Can we do the same as https://github.com/cerner/terra-framework/pull/2096/files#diff-f7d0eebfcee8de5c7b6734eb22dd4fbb2eacbcfd1ba69389b7ca4d7ae4e21d71R11 for all the places this line is added?

Reply:
Updated [here](https://github.com/cerner/terra-framework/pull/2096/commits/087ab290d418c6c6d43eeb6be0e4edb731c5e0a1)